_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,246 @@
+
+But, now lets say we have a confounder C, which is the common cause of both X and Y. And this is not observed. Now, we can change the distribution of C such that P(X,Y) is unchanged but we get different $$ P(Y \mid do(X)) $$
+$$ P(Y \mid do(X)) $$ can also be written as $$ P(y \mid \hat{x}) $$
+
+#### Back-Door Criterion 
+A set of variables Z satisfies the **back-door criterion** relative to an ordered pair of variables $$ (X_i, X_j) $$ if
+
+* No node in Z is a descendant of $$ X_i $$
+* Z blocks every path between $$ X_i $$ and $$ X_j $$ that contains an arrow into $$X_i$$ (Hence the name back-door)
+
+If the set of nodes Z follows these two conditions then $$ P (y \mid \hat{x}) = \sum_z P(y \mid x,z)P(z) $$
+Same as we saw earlier in the case of kidney stone treatment. And thus $$ P(y \mid \hat{x}) $$ is now identifiable because of this back-door adjustment.
+
+
+#### Front-Door Criterion
+A set of variables Z satisfies the front-door criterion relative to an ordered pair of variables $$ (X_i, X_j) $$ if
+* Z intercept all directed paths from X to Y (hence the name front-door)
+* There is no back-door path from X to Z 
+* All back-door paths from Z to Y are blocked by X
+If the set of nodes Z follows these two conditions then
+
+$$
+P(y \mid \hat{x}) = \sum_z P(z \mid x)\sum_{x'}P(y \mid x',z)P(x')
+$$
+
+
+
+####  Unification
+
+A sufficient condition under which the causal effect $$ P(y | do(x)) $$ is identifiable is that there exist no bi-directed path between X and any of its children. For more information, this is presented in the paper "A General Identification Condition for Causal Effect".
+
+#### Propensity Score
+The Average Causal Effect (ACE), defined as: 
+$$ 
+\mathbb{E} \left[ Y \mid do(x) \right] - \mathbb{E} \left[ Y \mid do(x') \right] 
+$$
+
+
+Given that $$ P(Y \mid do(x)) = \sum_{c} P(Y \mid x,c)P(c) $$ might not share the same $$ P(c) $$, one solution for an estimator of the ACE can be through direct matching. If $$ C $$ is a high dimensional space one can use the propensity score to achieve unconfoundedness and estimate ACE.
+### Causal Discovery
+
+Causal discovery relies on two assumptions:
+* **Causal Markov condition**: Each variable is independent of its non-descendants given its parents.
+
+* **Faithfulness condition**: All observed (conditional) independencies are captured in the causal graph. Roughly speaking, faithfulness allows you to go back from independence to causal graph.
+
+The above two conditions allows us to determine a list of candidate causal structures, with the same causal skeleton. Following this, we need to determine the causal direction. This step involves finding v-structure and doing orientation propagation. 
+
+The following example as given in lecture for reasoning about independence conditions:
+
+Suppose you have four variables $$ X_1, X_2, X_3, X_4 $$ such that:
+$$
```

Comment:
```suggestion

$$
```
Add a newline so the equation is centered. Right now it renders like this:
![image](https://user-images.githubusercontent.com/2617555/53582760-1ce88880-3b4e-11e9-99b1-164d08443cd1.png)